src/transformers/models/big_bird/modeling_big_bird.py

```diff
@@ -436,6 +437,69 @@ def __init__(self, config, seed=None):
         self.key = nn.Linear(config.hidden_size, self.all_head_size, bias=config.use_bias)
         self.value = nn.Linear(config.hidden_size, self.all_head_size, bias=config.use_bias)
 
+        self.rand_attn_tables = [[] for _ in range(config.max_position_embeddings // config.block_size - 1)]
+        self.generate_rand_attn_tables(
+            self.max_seqlen, self.max_seqlen, self.block_size, self.block_size, self.num_random_blocks, 1024
+        )
```

Comment:
How fast is this? Because if it takes 5 mins, might not be worth doing unless the user specifies that it is what he wants WDYT? 
Take someone who will just run a bunch of inferences, generating on the fly might be faster? (I don't knoew, it's up to you!)

Reply:
In torchbench, we only run bigbird for **only one iteration**. It executes `_bigbird_block_rand_mask` 432 times. I write [an example](https://github.com/FindHao/ml_scripts/blob/main/cases/bigbird.py) to test the performance of this function. In default mode, the original `_bigbird_block_rand_mask` takes about 90ms, while the `generate_rand_attn_tables` and `_bigbird_block_rand_mask_fast` take about 45ms. The `generate_rand_attn_tables`  takes about 12ms.